docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server]]
+= {fleet-server}
+
+beta[]
+
+{fleet-server} is a component used to centrally manage {agent}s. It serves as a
+control plane for updating agent policies, collecting status information, and
+coordinating actions across agents.
+
+//QUESTION: I think it's worth breaking down in detail how data flows between
+// the agent, Fleet Server, Fleet, and the stack, but I might have some details
+// wrong here. Is this correct?
+
+{fleet-server} is the mechanism that connects {agent}s to {fleet} and handles
+all communication between an {agent}, {fleet}, {es}, and {kib}. {fleet-server}:
+
+* Receives status information from connected {agent}s
+* Retrieves agent policies from {fleet} and stores them in {es}
+* Ships agent policies to all connected {agent}s
+* Handles communication with {es} and {kib}
+
+image:images/fleet-server-communication.png[Fleet Server handles communication between {agent}, {fleet}, {es}, and {kib}]
+
+{fleet-server} runs as a separate service alongside {agent}. The agent uses a
+special policy that governs how {fleet-server} connects to other systems and
+manages agent requests. In large scale self-managed deployments or on {ess-product}[hosted {ess}] on {ecloud}, {fleet-server} is typically run as a dedicated {agent} communication host,
+but you can optionally use it for data collection on self-managed clusters. For
+more details, refer to <<fleet-server-deployment>>.
+
+[discrete]
+[[fleet-server-compatibility]]
+== Compatibility
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later (our {ess-product}[hosted {ess}] on {ecloud} or
+a self-managed cluster)
+* {ece} 2.10 or later
+
+{fleet-server} is supported on x64 architectures only.
+
+[discrete]
+[[add-fleet-server]]
+== Add a {fleet-server}
+
+Before using {fleet} for central management, {fleet-server} must be running. The
+steps for running {fleet-server} on our {ess-product}[hosted {ess}] on {ecloud}
+are different from the steps for running it on a self-managed cluster.
+
+//TODO: Collapse this content into a tabbed widget after the steps are working.
+
+include::{tab-widgets}/add-fleet-server/widget.asciidoc[]
+
+
+Now you're ready to add {agent}s to your host systems. To learn how, see
+<<add-agent-to-fleet>>.
+
+//QUESTION: I think we probably want users to run Fleet Server as a
```

Comment:
By default we should encourage install but also tell them "somehwere" is an other option.

Reply:
Added inline as a TODO comment.